### PR TITLE
Implement OrganizerUI : Edit Properties

### PR DESCRIPTION
**Pull request by AlexisTabin**
OrganizerUI is almost the same as AttendeeUI with just a few differences : 

1. Properties can be edited through a Edit Button (pencil)
2. In properties edit mode, the organizer can confirm/cancel his changes
3. In properties edit mode, the organizer can add or delete witnesses

- [x] Add Pencil Button
- [x] Change Properties Fields from TextView to EditText
- [x] Add Confirm/Cancel Buttons
- [x] Add possibility to add witness
- [x] Add possibility to delete witness
- [x] When adding witness, a camera display with a prompt should appears
- [x] Test all above

### CODE REVIEW

File: app/src/main/res/drawable/ic_baseline_delete_24.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
```
Comment (celinecamacho) [inline app/src/main/res/drawable/ic_baseline_delete_24.xml:2]: The dimension values should be in the dimens.xml file, same for the ones in ic_baseline_edit_24.xml and event_category_layout.xml

File: app/src/main/java/com/github/dedis/student20_pop/utility/ui/AttendeeExpandableListViewEventAdapter.java [hunk 2]
```diff
@@ -6,6 +6,7 @@
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.BaseExpandableListAdapter;
+import android.widget.ImageButton;
```
Comment (celinecamacho) [inline app/src/main/java/com/github/dedis/student20_pop/utility/ui/AttendeeExpandableListViewEventAdapter.java:9]: I think this import is not used

File: app/src/androidTest/java/com/github/dedis/student20_pop/ui/OrganizerFragmentTest.java [hunk 3]
```diff
@@ -59,6 +64,93 @@ public void canCancelAddEvent() {
...
+        onView(withId(R.id.edit_button)).perform(click());
+        onData(anything()).inAdapterView(withId(R.id.witness_edit_list))
+                .atPosition(0).onChildView(withId(R.id.image_button_delete_witness)).check(matches(isDisplayed())).perform(click());
+        onView(withText("Delete ?")).check(matches(isDisplayed()));
+        onView(withText(R.string.button_confirm)).perform(click());
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    @Ignore("TODO : Is it right to allow empty string for a lao's name ?")
+    public void confirmNullLaoTitleThrowsException() {
```
Comment (celinecamacho) [inline app/src/androidTest/java/com/github/dedis/student20_pop/ui/OrganizerFragmentTest.java:140]: A LAO with an empty name is not a problem in our implementation, maybe we can change that on another issue

File: app/src/androidTest/java/com/github/dedis/student20_pop/ui/OrganizerFragmentTest.java [hunk 4]
```diff
@@ -59,6 +64,93 @@ public void canCancelAddEvent() {
...
+
+
+    @Test
+    @Ignore("TODO : Check that scanning a Witness QR code adds witness to witness list")
+    public void canAddWitness() {
+    }
+
     @Test
     @Ignore("TODO : Check that the corresponding Fragment has been launched")
     public void canLaunchCreateMeetingEventFragment() {
```
Comment (celinecamacho) [inline app/src/androidTest/java/com/github/dedis/student20_pop/ui/OrganizerFragmentTest.java:156]: Is it possible to test this or not yet?

**Review by celinecamacho — CHANGES_REQUESTED** [review]
Good, just some questions for the tests and hard coded values in xml files

File: app/src/main/res/drawable/ic_baseline_delete_24.xml (hunk 1 above)
Comment (AlexisTabin) [reply app/src/main/res/drawable/ic_baseline_delete_24.xml:2]: Done

File: app/src/main/java/com/github/dedis/student20_pop/utility/ui/AttendeeExpandableListViewEventAdapter.java (hunk 2 above)
Comment (AlexisTabin) [reply app/src/main/java/com/github/dedis/student20_pop/utility/ui/AttendeeExpandableListViewEventAdapter.java:9]: Removed

File: app/src/androidTest/java/com/github/dedis/student20_pop/ui/OrganizerFragmentTest.java (hunk 3 above)
Comment (AlexisTabin) [reply app/src/androidTest/java/com/github/dedis/student20_pop/ui/OrganizerFragmentTest.java:140]: It would be weird a LAO with an empty name, no ?

Comment (celinecamacho) [reply app/src/androidTest/java/com/github/dedis/student20_pop/ui/OrganizerFragmentTest.java:140]: Of course, I agree with you. But this should be modified and tested in another issue

Comment (AlexisTabin) [reply app/src/androidTest/java/com/github/dedis/student20_pop/ui/OrganizerFragmentTest.java:140]: Okay I was not sure with your first comment. I'll remove this test and open an issue

**Review by jbmichel14 — APPROVED** [review]
All good

**Review by celinecamacho — APPROVED** [review]

**Pull request merged**